apps/next/src/pages/Settings/components/NetworkManagement/components/NetworkDetailsFlow.tsx

```diff
@@ -1,17 +1,120 @@
-import { PageTopBar } from '@/components/PageTopBar';
-import { Stack } from '@avalabs/k2-alpine';
+import { useState } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import { CustomRpcHeadersManager } from './NetworkForm/CustomRpcHeadersManager';
+import { RpcUrlResetConfirmation } from './Confirmations/RpcUrlResetConfirmation';
+import { toast } from '@avalabs/k2-alpine';
+import { useTranslation } from 'react-i18next';
+import { NetworkUpdateConfirmation } from './Confirmations/NetworkUpdateConfirmation';
+import { EditNetworkFormTab } from './NetworkForm/types';
+import { NetworkDetails } from './NetworkDetails';
+import { NetworkDeleteConfirmation } from './Confirmations/NetworkDeleteConfirmation';
+import { NetworkNotFound } from './NetworkNotFound';
+import { useEditNetwork } from '../hooks/useEditNetwork';
+
+type NetworkDetailsParams = {
+  networkId: string;
+};
 
 export const NetworkDetailsFlow = () => {
-  return (
-    <Stack
-      height="100cqh"
-      width={1}
-      bgcolor="background.backdrop"
-      overflow="hidden"
-      sx={{ position: 'relative' }}
-    >
-      <PageTopBar showBack={true} />
-      <div>NetworkDetailsFlow</div>;
-    </Stack>
-  );
+  const history = useHistory();
+  const { t } = useTranslation();
+  const { networkId } = useParams<NetworkDetailsParams>();
+
+  const [tab, setTab] = useState<EditNetworkFormTab>('details');
```

Comment:
Why is this called a `tab`? I inferred there is a tab bar/tab panel involved, but nope :)
How about `view`?